src/test/test-taint.c

```diff
@@ -1,10 +1,12 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include "manager.h"
 #include "taint.h"
 #include "tests.h"
 
 TEST(taint_string) {
-        _cleanup_free_ char *a = taint_string();
+        Manager m = {};
```

Comment:
i feel very uncomfortable with this initialization, as the structure contains various fd fields which should not be initialized to zero, because zero is a valid fd...